### ISSUE TITLE
user login

### ISSUE THREAD

**Issue by cruoti** [discapp/discapp#26]
I have still not gotten anything to work with AWS Cognito for the login.  I have spent the majority of my time working the JavaScript of the individual pages.  ...  :(

We need to get this setup so that the user actually means something.

**Comment by mrossi113** [comment]
@cruoti AWS Cognito might not be the one service to work with Google login. From reading I think I found solution AWS mobile hub.
http://docs.aws.amazon.com/mobile-hub/latest/developerguide/create-google-project.html

**Comment by cruoti** [comment]
Ok, I'll have to check it out.  I've just about got the interactivity of the dashboard and chart/table set up.  The table updates, just working on the chart.

**Comment by mrossi113** [comment]
Yeah I just tested out the AWS mobile hub and only for ios and android so I'm going back to cognito again.

**Comment by cruoti** [comment]
well that's too bad.  Good news is that I finished making the chart and table update based on the dropdown in the dashboard.  It's up on EB.

**Comment by cruoti** [comment]
I've finished the interactions between the dashboard and the add scores page.  Everything is looking pretty good at the moment.  I'm thinking I need to help Kian with the database and get that hooked up this evening.  After that I can try to look back at this sign in.

**Comment by mrossi113** [comment]
My main focus tonight will be this. Than removing the links on index.html.
